packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_remove_oracle.test.ts

```diff
@@ -0,0 +1,75 @@
+import { MasterNodeRegTestContainer, GenesisKeys } from '@defichain/testcontainers'
+import { getProviders, MockProviders } from '../provider.mock'
+import { P2WPKHTransactionBuilder } from '../../src'
+import { calculateTxid, fundEllipticPair, sendTransaction } from '../test.utils'
+import { WIF } from '@defichain/jellyfish-crypto'
+
+const container = new MasterNodeRegTestContainer()
+let providers: MockProviders
+let builder: P2WPKHTransactionBuilder
+
+beforeAll(async () => {
+  await container.start()
+  await container.waitForReady()
+  await container.waitForWalletCoinbaseMaturity()
+
+  providers = await getProviders(container)
+  providers.setEllipticPair(WIF.asEllipticPair(GenesisKeys[GenesisKeys.length - 1].owner.privKey))
+  builder = new P2WPKHTransactionBuilder(providers.fee, providers.prevout, providers.elliptic)
+
+  // Prep 1000 DFI Token for testing
+  await container.waitForWalletBalanceGTE(1001)
+})
+
+afterAll(async () => {
+  await container.stop()
+})
+
+describe('remove oracle', () => {
+  beforeEach(async () => {
+    await container.waitForWalletBalanceGTE(1)
+  })
+
+  it('should appoint and then remove oracle', async () => {
+    // Fund 10 DFI UTXO
+    await fundEllipticPair(container, providers.ellipticPair, 10)
+    await providers.setupMocks() // required to move utxos
+
+    // Appoint Oracle
+    let script = await providers.elliptic.script()
```

Comment:
```suggestion
    const script = await providers.elliptic.script()
```

keep it const for better code writing, less mutable code = less errors.